config/config.go

```diff
@@ -333,6 +334,9 @@ type UIDefaults struct {
 // KialiFeatureFlags available from the CR
 type KialiFeatureFlags struct {
 	IstioInjectionAction bool       `yaml:"istio_injection_action,omitempty" json:"istioInjectionAction"`
+	IstioUpgradeAction   bool       `yaml:"istio_upgrade_action,omitempty" json:"istioUpgradeAction"`
+	IstioCurrentRevision string     `yaml:"istio_current_revision,omitempty" json:"istioCurrentRevision"`
```

Comment:
One comment, as we have splitted the logical of the config in several sections, my thoughts:

- IstioLabels -> InjectionLabelRev. Correct. Typically it will be a fix label similar as app/version used by Istio.
- KialiFeatureFlags -> IstioUpgradeAction. Correct. This should section show enable/disable the feature. (I have no idea who approved the UIDefaults, that one doesn't belong here).
- I'd move IstioCurrentRevision -> IstioUpgradeRevision to the IstioConfig, as it's where I'd look for when I want to personalize something related to my Istio mesh (i.e. I'm going to upgrade between 1.9 and 1.10, so I'd look here).

But anyway, +1 to enable this feature from the config. 

Naming is complex as everyone wants to put his/her criteria, the idea here is to try to put together the same info in the sections that user would look for.

Reply:
> * I'd move IstioCurrentRevision -> IstioUpgradeRevision to the IstioConfig, as it's where I'd look for when I want to personalize something related to my Istio mesh (i.e. I'm going to upgrade between 1.9 and 1.10, so I'd look here).

Yes. To be clear, [IstioConfig](https://github.com/kiali/kiali/blob/v1.36.0/config/config.go#L170-L181) is this section in the CR: https://github.com/kiali/kiali-operator/blob/v1.36.0/deploy/kiali/kiali_cr.yaml#L549-L569

This is also what I recommended in my earlier comment https://github.com/kiali/kiali/pull/4122#discussion_r657958774 : `consider putting these in external_services.istio along with the other istio-specific things`